src/modules/dashboard/components/ExpenditurePage/Payments/Payments.css

```diff
@@ -0,0 +1,99 @@
+@value paddingRightLeft: 54px;
+
+.paymentContainer {
+  padding-bottom: 80px;
+}
+
+.recipientContainer {
+  padding: 0 paddingRightLeft;
+}
+
+.payments {
+  margin-bottom: 18px;
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  line-height: 18px;
+  color: var(--dark);
+}
+
+.recipientLabel {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 10px;
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  line-height: 18px;
+  color: var(--dark);
+}
+
+.singleRecipient {
+  padding-bottom: 4px;
+  position: relative;
+}
+
+.signWrapper {
+  height: 30px;
+  width: 30px;
+  position: absolute;
+
+  /* negative margin is used to set the position of the button,
+  which collapse and expand singe recipient fields,
```

Comment:
It's a nitpick but there are types here. I assume it should be `which collapses and expands single recipient fields`?